samples/Basic_Serial/files/Readme.md

```diff
@@ -0,0 +1,322 @@
+# Sming
```

Comment:
Maybe it would be better to change the makefile of the app to copy the Readme file, if not present, from the Sming root directory and store it under `files`.